dev_guide/deployments.adoc

```diff
@@ -77,14 +86,21 @@ spec:
           name: "origin-ruby-sample:latest"
   strategy: <5>
     type: "Rolling"
+  paused: false <6>
+  revisionHistoryLimit: 2 <7>
+  minReadySeconds: 0 <8>
+
 ----
 
-<1> The replication controller template named `frontend` describes a simple Ruby application.
-<2> There will be 5 replicas of `frontend` by default.
-<3> A xref:config-change-trigger[configuration change trigger] causes a new deployment to be created any time the replication controller template changes.
-<4> An xref:image-change-trigger[image change trigger] trigger causes a new deployment to be
-created each time a new version of the `origin-ruby-sample:latest` image repository is available.
-<5> The xref:rolling-strategy[Rolling strategy] is the default and may be omitted.
+<1> The pod template of the `frontend` deployment configuration describes a simple Ruby application.
+<2> There will be 5 replicas of `frontend`.
+<3> A xref:config-change-trigger[configuration change trigger] causes a new replication controller to be created any time the pod template changes.
+<4> An xref:image-change-trigger[image change trigger] trigger causes a new replication controller to be
+created each time a new version of the `origin-ruby-sample:latest` image stream tag is available.
+<5> The xref:rolling-strategy[Rolling strategy] is the default way of deploying your pods. May be omitted.
+<6> If true, `paused` makes inert your deployment configuration and subsequent changes to the pod template or new images will not run new deployments.
```

Comment:
The existing running deployment will not be paused. Pod template or image changes cover all the spectrum of the changes on which we run deployments currently.